ui/src/components/home/ProfileContainerDesktop.tsx

```diff
@@ -0,0 +1,387 @@
+import axios from 'axios';
+import { FC, ReactElement, useState, useEffect, useCallback } from 'react';
+import { useForm } from 'react-hook-form';
+import { useSelector } from 'react-redux';
+
+import { Box, Button, Divider, InputAdornment, Stack, TextField, Typography } from '@mui/material';
+import ArrowBackIosNewIcon from '@mui/icons-material/ArrowBackIosNew';
+import CreateIcon from '@mui/icons-material/Create';
+import IconButton from '@mui/material/IconButton';
+import PersonIcon from '@mui/icons-material/Person';
+import Visibility from '@mui/icons-material/Visibility';
+import VisibilityOff from '@mui/icons-material/VisibilityOff';
+
+function HomeProfile(props: any) {
```

Comment:
we want to use proper typescript so can you define the props? so like 
```
interface HomeProfileProps {
onBack: ...
}
```

you can see my PR on how i define and put props in